### PR TITLE
Saves as snapshots during file mutation and deletes files after exit

### PR DESCRIPTION
**Pull request by akd13**
I didn't implement a method to check if the data is being mutated as it seems to be handled by the app itself - snapshots were only created during the add, delete and edit commands. However if you guys discover a bug let me know

### CODE REVIEW

File: src/main/java/seedu/multitasky/storage/StorageManager.java [hunk 1]
```diff
@@ -14,6 +14,7 @@
 import seedu.multitasky.model.ReadOnlyEntryBook;
 import seedu.multitasky.model.UserPrefs;
 
+//@@author A0132788U
```
Comment (mattheuslee) [inline src/main/java/seedu/multitasky/storage/StorageManager.java]: The collator for the code takes everything from the //@@author onwards until it finds another //@@author, so if you have one at the beginning of the file there's no need for the others.

**Review by mattheuslee — APPROVED** [review]

**Review by kevinLamKB — APPROVED** [review]

File: src/main/java/seedu/multitasky/storage/StorageManager.java (hunk 1 above)
Comment (akd13) [reply src/main/java/seedu/multitasky/storage/StorageManager.java]: Ok noted!

**Comment by mattheuslee** [comment]
Is it possible to also make a snapshot for the `clear` command? That seems like something that you'd want to undo.

**Comment by akd13** [comment]
@mattheuslee hmm weird I get a snapshot on my laptop when I execute clear

**Comment by mattheuslee** [comment]
@akd13 oh okay then for now it's alright then, as long as it does that every time. we can figure out why later.

**Review by ChuaPingChan — APPROVED** [review]

**Pull request merged**